pkg/cloudprovider/providers/gce/gce_loadbalancer_external.go

```diff
@@ -456,6 +423,40 @@ func (gce *GCECloud) DeleteExternalTargetPoolAndChecks(name, region, clusterID s
 	return nil
 }
 
+// verifyUserRequestedIP checks the user-provided IP  to see whether it can be
```

Comment:
Do you mean to renamed `ownedByUser` to `isUserOwnedIP`? Done. Although it's not used anywhere, and it's just here for clarity.